pkg/exchange/okex/parse.go

```diff
@@ -15,41 +15,42 @@ import (
 	"github.com/c9s/bbgo/pkg/types"
 )
 
-func parseWebSocketEvent(str []byte) (interface{}, error) {
-	v, err := fastjson.ParseBytes(str)
+func parseWebSocketEvent(in []byte) (interface{}, error) {
+	var e WsEvent
+
+	err := json.Unmarshal(in, &e)
 	if err != nil {
 		return nil, err
 	}
 
-	if v.Exists("event") {
-		return parseEvent(v)
-	}
-
-	if v.Exists("data") {
-		return parseData(v)
+	switch {
+	case e.IsOp():
+		parseEvent(&e)
+	case e.IsTopicEvent():
+		parseData(&e)
 	}
```

Comment:
add default case to return unhandled error

Reply:
already fixed in latest commit